<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Register</class>
 <widget class="QWidget" name="Register">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Register</string>
  </property>
  <widget class="QWidget" name="titleWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>50</height>
    </rect>
   </property>
   <property name="titleskin" stdset="0">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="bodyWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>400</width>
     <height>250</height>
    </rect>
   </property>
   <property name="bottomskin" stdset="0">
    <bool>true</bool>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>28</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>您的部门：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>164</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>您的寸照：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="nameLineEdit">
    <property name="geometry">
     <rect>
      <x>182</x>
      <y>80</y>
      <width>151</width>
      <height>28</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>请输入员工姓名</string>
    </property>
   </widget>
   <widget class="QPushButton" name="selectPictureBtn">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>140</y>
      <width>41</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QComboBox" name="employeeDepBox">
    <property name="geometry">
     <rect>
      <x>182</x>
      <y>20</y>
      <width>151</width>
      <height>28</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>人事部</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>研发部</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>市场部</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="headLabel">
    <property name="geometry">
     <rect>
      <x>201</x>
      <y>142</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>  员工寸照</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>87</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>您的姓名：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addBtn">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>200</y>
      <width>151</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>注册</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
